src/statement.cc

```diff
@@ -302,20 +302,22 @@ bool Statement::Bind(const Parameters & parameters) {
 
             switch (field->type) {
                 case SQLITE_INTEGER: {
-                    status = sqlite3_bind_int(_handle, pos,
+                    status = sqlite3_bind_int64(_handle, pos,
                         ((Values::Integer*)field)->value);
                 } break;
                 case SQLITE_FLOAT: {
                     status = sqlite3_bind_double(_handle, pos,
                         ((Values::Float*)field)->value);
                 } break;
                 case SQLITE_TEXT: {
-                    status = sqlite3_bind_text(_handle, pos,
+                    status = sqlite3_bind_text64(_handle, pos,
```

Comment:
Ref https://github.com/TryGhost/node-sqlite3/blob/master/src/statement.h#L46 for the len field.
I also checked out the last stable of sqlite and confirmed that sqlite3_bind_text passes SQLITE_UTF8 as the last parameter.